doc/src/parser/item.rs

```diff
@@ -116,7 +116,11 @@ impl ParseItem {
     /// Format the item's filename.
     pub fn filename(&self) -> String {
         let prefix = match self.source {
-            ParseSource::Contract(_) => "contract",
+            ParseSource::Contract(ref c) => match c.ty {
+                ContractTy::Contract(_) | ContractTy::Abstract(_) => "contract",
```

Comment:
I think it might be good to also distinguish between regular and abstract contracts, since one is meant to be deployed and the other is not. Wdyt @PaulRBerg?

Reply:
agreed

Reply:
what should the ident for abstract contracts be?
just "abstract"?

Reply:
"abstract" or "abstractContract" both seem ok with me, no strong preference

Reply:
Sorry guys, you moved way quicker than I expected!

Agree with the call to also distinguish abstract contracts. My choice would have been `abstract`, for brevity.